libraries/app/api.cpp

```diff
@@ -151,7 +151,7 @@ namespace graphene { namespace app {
              {
                 auto block_num = b.block_num();
                 auto& callback = _callbacks.find(id)->second;
-                fc::async( [capture_this,this,id,block_num,trx_num,trx,callback]() {
+                fc::async( [capture_this,id,block_num,trx_num,trx,callback]() {
```

Comment:
I have been working on this line of code for another PR, and can verify that the change above causes no side effects. Unfortunately it doesn't fix my problem, but that is another story...

Reply:
I want to hear more discussion about this. @pmconrad ?

Reply:
Removing "this" shouldn't make a difference. Nevertheless I think we should wait for John's results from #813 before changing this.